mobile/App.tsx

```diff
@@ -1,20 +1,79 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Image } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { HomeScreen } from './src/screens/HomeScreen';
+import { Settings } from './src/screens/Settings';
+import { ChatScreen } from './src/screens/ChatScreen';
+
+const Tab = createBottomTabNavigator();
 
 const App: React.FC = () => {
   return (
-    <View style={styles.container}>
-      <Text>TODO</Text>
-    </View>
+    <NavigationContainer>
+      <Tab.Navigator
+        initialRouteName="Home"
+        screenOptions={({ route }) => ({
+          tabBarIcon: ({ focused }) => {
+            let icon;
+            const tint = focused ? '#E89228' : '#B8B8B8';
+            let width;
+            let height;
```

Comment:
Let's define these styles in the style sheet below, just to keep things a little cleaner.